libs/utils/android/workloads/uibench.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, test_env):
         # Set of output data reported by UiBench
         self.db_file = None
 
-    def run(self, out_dir, test_name, duration_s, collect=''):
+    def run(self, out_dir, test_name, duration_s, collect='', action=''):
```

Comment:
Done. Now part of a dictionary with 'sensible' default actions that by default will be performed when given a test that matches the key.